### DISCUSSION TITLE
timelapse using bridge

### DISCUSSION THREAD

**Discussion by letrain02** [mrlt8/docker-wyze-bridge#757] [Ideas]
anyway to get a snapshot saved on intervals to individual folders with dates/times like recording is able to do? I'd like to basically have a timelapse saved from the bridge. i'm currently using motion eye to save an image 4 times an hour. it is the only reason i have motioneye and i'd like to eliminate it.

**Comment by mrlt8** [comment]
Added a `SNAPSHOT_FORMAT` option to the[ dev branch](https://github.com/mrlt8/docker-wyze-bridge/commit/1a0c9d48671152a27fd80954fb862a7c6ca2b896) that should work with the existing `SNAPSHOT` option to specify the output format. So it should be possible with something like `SNAPSHOT=RTSP900` and formatting your own output with something like `SNAPSHOT_FORMAT={cam_name}/%Y-%m-%d/%H-%M.jpg`

**Reply by letrain02** [reply]
awesome. 

would also be in the dev-hw? dockerhub looks like they both updated at same time. currently on edge-hw, but thats not that big of a step.

thanks!

**Reply by mrlt8** [reply]
yes, dev changes should be reflected in the dev-hw builds. 

FYI, the HW builds use a different version of ffmpeg so there could be some potential differences due to that.

**Comment by letrain02** [comment]
yes. been using edge-hw for a while.. anyway to seperate recordings into date folders? i've been trying but it either just make files in the cam_name folder, or does nothing...

**Reply by mrlt8** [reply]
Unfortunately, the strftime option in ffmpeg doesn't seem to support directories when segmenting the recorings.

**Comment by letrain02** [comment]
i noticed you added snapshot_format to latest, however i can't get it working. works fine in dev though.

**Reply by mrlt8** [reply]
Seems to be working for me. Do you have the `SNAPSHOT` interval set? something like `SNAPSHOT=RTSP60`
